style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  width: 320px;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  background-image: url(assets/images/image_geometry_1.svg);
+  background-repeat: no-repeat;
+  background-size: 111px 74px;
+  background-color: #fff;
+  margin: 0 auto;
+  padding: 20px;
+  align-items: center;
+}
```

Comment:
- [Optional] 🛑🛑 Dear @christianonoh, Please notice your design needs more adjustment to make it look like the one on the figma design. Your `button`  needs to take most of the width and be aligned to the center, and the hamburger button needs to be on the right side. 

### Your design 
![Screenshot 2023-02-17 200835](https://user-images.githubusercontent.com/69077061/219721447-557e8e73-3ea8-4653-9ae8-9b5300665920.png)
### Figma design 
![Screenshot 2023-02-17 200924](https://user-images.githubusercontent.com/69077061/219721467-b717377b-f99e-4450-9c83-c08c0c6e4395.png)